easyrest/models/meta.py

```diff
@@ -49,21 +50,47 @@ def __repr__(self):
         print s
         return ""
 
-    def as_dict(self, exclude=[], include=[]):
-        """Converts model into python dictionary
+    def as_dict(self, exclude=[], include=[], with_relations=[], _isNested=True):
+        """Converts model into python dictionary.
+        Also support relation unpack to one depth
+        To unpack relationship specify its name in with_relations
+        Example: model.as_dict(with_relations=["<relation1>", "<relation2>"])
+        Pls don`t reasing _isNested it uses to limit recursive depth
+        Args:
+            exclude: (list) (str) list of fields to be excluded more priority then include
+            include: (list) (str) list of fields to be included
+            with_relations: (list) (str) list of relationships to include
+            _isNested: (bool) system variable to limit recursion depth
         Returns:
             dictionary
             {
-                [prop.name = prop.value, ]
+                [prop.name = prop.value, ... ]
             }
             if it finds datetime object converts it to isoformat.
 
         """
         data = {}
+        rel_data = {}
+        relationships = self.__mapper__.relationships.keys()
+        for name in relationships:
+            if not _isNested or name not in with_relations:
```

Comment:
if `_isNested` is True, it will always continue and this loop has no sense, I'd move check for this condition above and do not perform any relationships tasks at all if it is True.

Reply:
I see, i will also get rid of loop because here i explicitly pass relationship names so i don't need nor loop, nor relationship.keys()